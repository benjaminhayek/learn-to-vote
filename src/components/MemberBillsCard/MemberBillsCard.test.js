import React from 'react';
import { MemberBillsCard } from './index';
import { shallow } from 'enzyme';

describe('MemberBillsCard', () => {
  let wrapper;
  let bills;
  
  beforeEach(() => {
    bills = {comittee: 'Education', title: 'Bill title', url: 'www.gov.com'}
    wrapper = shallow(<MemberBillsCard bills={bills}/>);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})