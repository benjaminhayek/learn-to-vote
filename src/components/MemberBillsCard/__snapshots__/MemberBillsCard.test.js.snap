// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MemberBillsCard should match the snapshot 1`] = `
<div
  className="bills-container"
>
  <h1
    className="committee"
  >
    Committee: 
  </h1>
  <h2
    className="bill-title"
  >
    Title: 
    Bill title
  </h2>
  <a
    className="url"
    href="www.gov.com"
  >
    Click here for more info
  </a>
</div>
`;
